### PR TITLE
Translate part-two

### PR DESCRIPTION
**Pull request by andriawan**
Dear maintainers @cendekia, @coroo, @dennyfauzan. I have finished translating part two. Please review it and give me some feedback. Thanks

### CODE REVIEW

File: docs/tutorial/part-two/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Introduction to Styling in Gatsby
+title: Pengenalan Styling di Gatsby
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Title change with “judul”

Comment (coroo) [reply docs/tutorial/part-two/index.md]: Also, please add italic format in styling

File: docs/tutorial/part-two/index.md [hunk 2]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
 
 -->
 
-Welcome to part two of the Gatsby tutorial!
+Selamat datang di bagian dua dari tutorial Gatsby!
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Bagian kedua, i think its better use that word

File: docs/tutorial/part-two/index.md [hunk 3]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
 
-## What's in this tutorial?
+## Apa yang ada di dalam tutorial ini?
 
-In this part, you're going to explore options for styling Gatsby websites and dive deeper into using React components for building sites.
+Pada bagian ini, Anda akan menjelajahi pilihan untuk _styling_ situs web Gatsby dan menyelam lebih dalam tentang penggunaan komponen React untuk membangun situs.
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: menyelam lebih dalam, i think we can use another text like “menyelami” or “mempelajari lebih dalam”

File: docs/tutorial/part-two/index.md [hunk 4]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
```
Comment (coroo) [inline docs/tutorial/part-two/index.md:22]: Please use italic in english language context

File: docs/tutorial/part-two/index.md [hunk 5]
```diff
@@ -103,67 +103,67 @@ Your project's file structure should now look like this:
 ├── gatsby-browser.js
 ```
 
-> 💡 What is `gatsby-browser.js`? Don't worry about this too much and for now, just know that `gatsby-browser.js` is one of a handful of special files that Gatsby looks for and uses (if they exist). Here, the naming of the file **is** important. If you do want to explore more now, check out [the docs](/docs/browser-apis/).
+> 💡 Apa itu `gatsby-browser.js`? Jangan terlalu khawatir tentang ini untuk sekarang, pahami saja bahwa `gatsby-browser.js` merupakan salah satu dari beberapa berkas khusus yang Gatsby cari dan gunakan (jika ada). Disini, penamaan berkas **merupakan** sesuatu yang penting. Jika Anda ingin mendalami lebih jauh sekarang, Periksa [dokumentasi](/docs/browser-apis/).
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Please use lower case for last word “Periksa”

File: docs/tutorial/part-two/index.md [hunk 6]
```diff
@@ -103,67 +103,67 @@ Your project's file structure should now look like this:
...
 
-2. Import your recently-created stylesheet in the `gatsby-browser.js` file:
+2. Impor stylesheet yang baru saja Anda buat ke dalam berkas `gatsby-browser.js`:
 
 ```javascript:title=gatsby-browser.js
-import "./src/styles/global.css"
+import './src/styles/global.css';
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: For code, please dont change it. Revert it to code from Gatsby please

File: docs/tutorial/part-two/index.md [hunk 7]
```diff
@@ -103,67 +103,67 @@ Your project's file structure should now look like this:
...
+> **CSS Module** merupakan berkas CSS yang mana semua nama kelas dan nama animasi
+> normalnya dicakup secara lokal.
 
-CSS Modules are very popular because they let you write CSS normally but with a lot more safety. The tool automatically generates unique class and animation names, so you don't have to worry about selector name collisions.
+CSS Modules sangat populer karena CSS Modules mengizinkan Anda menulis CSS secara normal namun dengan banyak keamanan. CSS Modules secara otomatis menghasilkan nama kelas dan animasi unik, jadi Anda tidak perlu khawatir tentang penamaan _selector_ yang sama.
 
-Gatsby works out of the box with CSS Modules. This approach is highly recommended for those new to building with Gatsby (and React in general).
+Gatsby berfungsi _out of the box_ dengan CSS Modules. Pendekatan ini sangat direkomendasikan untuk mereka yang awam dalam pengembangan menggunakan Gatsby (dan React pada umumnya).
 
 #### ✋ Build a new page using CSS Modules
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Please translate this line

File: docs/tutorial/part-two/index.md [hunk 8]
```diff
@@ -103,67 +103,67 @@ Your project's file structure should now look like this:
...
-First, create a new `Container` component.
+Pertama, buat komponen `Container` baru.
 
-1. Create a new directory at `src/components` and then, in this new directory, create a file named `container.js` and paste the following:
+1. Buat direktori baru pada `src/components` lalu, dalam direktori tersebut, buat berkas dengan nama `container.js` dan tempel kode berikut:
 
 ```javascript:title=src/components/container.js
-import React from "react"
-import containerStyles from "./container.module.css"
+import React from 'react';
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Please leave the code, dont change any of it

File: docs/tutorial/part-two/index.md [hunk 9]
```diff
@@ -103,67 +103,67 @@ Your project's file structure should now look like this:
...
+import containerStyles from './container.module.css';
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Please leave it

File: docs/tutorial/part-two/index.md [hunk 10]
```diff
@@ -103,67 +103,67 @@ Your project's file structure should now look like this:
...
 
 export default ({ children }) => (
   <div className={containerStyles.container}>{children}</div>
-)
+);
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Also please leave it

File: docs/tutorial/part-two/index.md [hunk 11]
```diff
@@ -103,67 +103,67 @@ Your project's file structure should now look like this:
...
 ```
 
-You'll notice you imported a css module file named `container.module.css`. Let's create that file now.
+Anda akan melihat Anda mengimpor file modul css bernama `container.module.css`. Mari kita buat file itu sekarang.
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Please choose between “berkas” and “file”. You use both of those word, its better to choose 1 of then

File: docs/tutorial/part-two/index.md [hunk 12]
```diff
@@ -172,35 +172,35 @@ You'll notice you imported a css module file named `container.module.css`. Let's
...
-You'll notice that the file name ends with `.module.css` instead of the usual `.css`. This is how you tell Gatsby that this CSS file should be processed as a CSS module rather than plain CSS.
+Anda akan melihat bahwa nama file diakhiri dengan `.module.css` alih-alih`.css` biasa. Ini adalah cara Anda memberi tahu Gatsby bahwa file CSS ini harus diproses sebagai modul CSS daripada CSS biasa.
 
-3. Create a new page component by creating a file at
+3. Buat komponen halaman baru dengan cara membuat berkas baru pada
    `src/pages/about-css-modules.js`:
 
 ```javascript:title=src/pages/about-css-modules.js
-import React from "react"
+import React from 'react';
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Leave the code

File: docs/tutorial/part-two/index.md [hunk 13]
```diff
@@ -236,35 +236,34 @@ In this section, you'll create a list of people with names, avatars, and short L
 }
 ```
 
-3. Import the new `src/pages/about-css-modules.module.css` file into the `about-css-modules.js` page you created earlier by editing the first few lines of the file like so:
+3. Impor berkas baru `src/pages/about-css-modules.module.css` ke dalam halaman `about-css-modules.js` yang sudah anda buat diawal dengan cara menyunting beberapa baris pertama pada berkas seperti:
 
 ```javascript:title=src/pages/about-css-modules.js
```
Comment (coroo) [inline docs/tutorial/part-two/index.md:241]: Leave the code

File: docs/tutorial/part-two/index.md [hunk 14]
```diff
@@ -236,35 +236,34 @@ In this section, you'll create a list of people with names, avatars, and short L
...
-import React from "react"
+import React from 'react';
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Leave the code

File: docs/tutorial/part-two/index.md [hunk 15]
```diff
@@ -236,35 +236,34 @@ In this section, you'll create a list of people with names, avatars, and short L
...
 
-Create a new `<User />` component inline in the `about-css-modules.js` page
-component. Modify `about-css-modules.js` so it looks like the following:
+Buat komponen `<User />` baru dibarisan dalam komponen halaman `about-css-modules.js`. Ubah `about-css-modules.js` menjadi seperti berikut:
 
 ```jsx:title=src/pages/about-css-modules.js
-import React from "react"
-import styles from "./about-css-modules.module.css"
-import Container from "../components/container"
+import React from 'react';
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: Leave the code

File: docs/tutorial/part-two/index.md [hunk 16]
```diff
@@ -295,39 +294,39 @@ export default () => (
...
+Gatsby mendukung hampir setiap opsi styling (jika belum ada plugin untuk opsi CSS favorit Anda, [silahkan berkontribusi!](/contributing/how-to-contribute/))
 
 - [Typography.js](/packages/gatsby-plugin-typography/)
 - [Sass](/packages/gatsby-plugin-sass/)
 - [JSS](/packages/gatsby-plugin-jss/)
 - [Stylus](/packages/gatsby-plugin-stylus/)
 - [PostCSS](/packages/gatsby-plugin-postcss/)
 
-and more!
+and banyak lagi!
```
Comment (coroo) [inline docs/tutorial/part-two/index.md]: And banyak lagi should be “dan banyak lagi”

**Review by coroo — CHANGES_REQUESTED** [review]
Good job bro! 
Please help us to finish some comment above. Thank you

File: docs/tutorial/part-two/index.md (hunk 1 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md]: got it

File: docs/tutorial/part-two/index.md (hunk 2 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md]: i agree

File: docs/tutorial/part-two/index.md (hunk 3 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md]: it sounds better for "mempelajari lebih dalam"

File: docs/tutorial/part-two/index.md (hunk 4 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md:22]: do you mean web ?

File: docs/tutorial/part-two/index.md (hunk 5 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md]: got it

File: docs/tutorial/part-two/index.md (hunk 6 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md]: I'm sorry...this is caused by my vscode internal linter

File: docs/tutorial/part-two/index.md (hunk 7 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md]: should we translate CSS modules? is it a name or a general meaning?

Comment (andriawan) [reply docs/tutorial/part-two/index.md]: i have no idea the meaning of out of the box perceptively

File: docs/tutorial/part-two/index.md (hunk 8 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md]: sorry this is my linter again

File: docs/tutorial/part-two/index.md (hunk 11 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md]: yes, my inconsistency...I'll fix it

File: docs/tutorial/part-two/index.md (hunk 16 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md]: got it

File: docs/tutorial/part-two/index.md (hunk 7 above)
Comment (coroo) [reply docs/tutorial/part-two/index.md]: Sorry my mistake, i mean line 147

File: docs/tutorial/part-two/index.md (hunk 4 above)
Comment (coroo) [reply docs/tutorial/part-two/index.md:22]: Sorry, i mean line 23

**Review by coroo — APPROVED** [review]
Thank you @andriawan .
Please dont forget to claim your reward.

**Comment by andriawan** [comment]
Okey, thank you @coroo and the others maintainers for letting me contribute to this project.

File: docs/tutorial/part-two/index.md (hunk 1 above)
Comment (andriawan) [reply docs/tutorial/part-two/index.md]: should we really translate title into "judul"? After i checked other translations, they don't translate it.

Comment (coroo) [reply docs/tutorial/part-two/index.md]: For now, i think we can leave it.

**Pull request merged**